### DISCUSSION TITLE
emp grenades create the big lightning effect thingy on your character if you pull the pin before throwing it

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#14267] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

i was testing the new grenade stuff and when i tried to pull pin and throw on an emp nade i noticed the particle was showing up on my character and not where the grenade explodes

### Reproduction steps

pull pin on emp grenade
throw grenade
witness lightning thingy on character

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Could not repro this. Particles show on the grenade when I throw it, particles show on character if I hold it until it explodes. Seems to function as intended?


https://github.com/FakeFishGames/Barotrauma/assets/8641689/b73ce366-981e-4810-a622-f7c837fab3fb

**Reply by LePiromano** [reply]
> Could not repro this. Particles show on the grenade when I throw it, particles show on character if I hold it until it explodes. Seems to function as intended?

nvm the issue was somethign else entirely i'm mad stupid, the particle generates on the submarine's center of mass
https://github.com/user-attachments/assets/aa4c4d88-993c-4f39-b0c3-adc2239ecc1a

**Comment by JeroonBosch** [comment]
Alright np, closing then
